src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -909,11 +943,11 @@ PropertyList Visitor::visitTypesafe(Parser::PropertyListNotEmptyContext* ctx) {
 }
 
 // ____________________________________________________________________________________
-VarOrTerm Visitor::visitTypesafe(Parser::VerbContext* ctx) {
+VarOrTerm Visitor::visit(Parser::VerbContext* ctx) {
   // TODO<qup42, joka921> Is there a way to make this visitAlternative in the
   // presence of the a case?
   if (ctx->varOrIri()) {
-    return visitTypesafe(ctx->varOrIri());
+    return visit(ctx->varOrIri());
```

Comment:
I think the TODO above can be removed. It think it is not easily possible, and I also don't think it would be worth it.